schema/card_schema.json

```diff
@@ -46,206 +124,348 @@
+                    "enum": [-2]
+                },
+                {
+                    "type": "integer",
+                    "minimum": 0
+                }
+            ],
             "type": "integer"
         },
         "skill_intellect": {
-            "minimum": -2,
+            "description": "Number of intellect icons on the card, or intellect skill for an investigator. -2 means `X` in the latter case.",
+            "oneOf": [
+                {
+                    "type": "integer",
+                    "enum": [-2]
+                },
+                {
+                    "type": "integer",
+                    "minimum": 0
+                }
+            ],
             "type": "integer"
         },
         "skill_wild": {
-            "minimum": 0,
+            "description": "Number of wild icons on the card, or wild skill for an investigator. -2 means `X` in the latter case.",
+            "oneOf": [
+                {
+                    "type": "integer",
+                    "enum": [-2]
+                },
+                {
+                    "type": "integer",
+                    "minimum": 1
+                }
+            ],
             "type": "integer"
         },
         "skill_willpower": {
-            "minimum": -2,
-            "type": "integer"
-        },
-        "sanity": {
+            "description": "Number of willpower icons on the card, or willpower skill for an investigator. -2 means `X` in the latter case.",
             "oneOf": [
                 {
-                    "minimum": -2,
-                    "type": "integer"
+                    "type": "integer",
+                    "enum": [-2]
                 },
                 {
-                    "type": "null"
+                    "type": "integer",
+                    "minimum": 0
                 }
-            ]
+            ],
+            "type": "integer"
         },
         "slot": {
+            "description": "Slot of an asset. Can be omitted for assets that don't have a slot.",
             "minLength": 1,
             "type": "string"
         },
+        "stage": {
+            "description": "Stage of the act or agenda. Required to be present for acts.",
+            "minimum": 0,
+            "type": "integer"
+        },
         "subname": {
+            "description": "Subname of the card.",
             "minLength": 1,
             "type": "string"
         },
         "subtype_code": {
-            "oneOf": [
-                {
-                    "minLength": 0,
-                    "type": "string"
-                },
-                {
-                    "type": "null"
-                }
-            ]
+            "description": "Subtype code of the card. Used to mark weaknesses.",
+            "minLength": 1,
+            "type": "string"
+        },
+        "tags": {
+            "description": "Card tags are used to accommodate special deck building rules that are hard to filter for otherwise.",
```

Comment:
```suggestion
            "description": "Card tags are used to accommodate special deck building rules that are hard to filter for otherwise. Tags include `fa` for cards that reference the [[Firearm]] trait, `hh` for cards that heal horror, `hd` for cards that heal damage, `pa` for cards with parley, `se` for cards that seal.",
```

Reply:
@drawntotheflame Maybe we should point to the readme here? Tags are changing a lot

Reply:
Yes, that seems reasonable to me!